### ISSUE TITLE
Unnamed players can't name themselves

### ISSUE THREAD

**Issue by Shadie116** [jasonrohrer/OneLife#960]

**Comment by cordy-123** [comment]
Just ask someone else to name you.

Anyone can name anyone else. 

People can even name a nameless Eve.

**Comment by Shadie116** [comment]
the goal of this is to name yourself if you're nameless

**Comment by jasonrohrer** [comment]
Yeah, you have to be named by someone else.  This is fundamental to the game.
